### ISSUE TITLE
Allow deletion of a "family" along with all learning data

### ISSUE THREAD

**Issue by gesslerpd** [schollz/find3#36]
With regard to https://cloud.internalpositioning.com, this would mean that family names should be kept secret.

**Comment by gesslerpd** [comment]
This will encourage the use of relatively secret family names. Right now anyone that knows a family name cannot delete data, but they can skew learning data to be incorrect, by sending bad learning data to the family.

**Comment by schollz** [comment]
> encourage the use of relatively secret family names

I think this is a good approach. That way everyone can just call their family "foo" and try it until someone else deletes it. 

Also, what about a requirement for a minimum length on the family name?

**Comment by gesslerpd** [comment]
Yeah sounds good to me! Maybe this feature could be API only `DELETE` method, so you can share the web interface link without worrying about people clicking buttons and deleting accidentally.
